source/fundamentals/logging.txt

```diff
@@ -0,0 +1,298 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+You will learn how to:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+This section gives background on the dependencies necessary to set up a
+logger and provides an example logger setup. 
+
+Background
+~~~~~~~~~~
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+Setting up a logger is optional. When you start your application the MongoDB
+Java driver looks for the ``slf4j-api`` package in your classpath. If the driver
+can't find the ``slf4j-api`` package, the driver logs the following warning with
+``java.util.logging`` and disables all further logging:
+
+.. code-block:: none
+
+   WARNING: SLF4J not found on the classpath.  Logging is disabled for the 'org.mongodb.driver' component
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
+connects the ``slf4j-api`` package with a logging framework. The example below shows
+how to bind the ``slf4j-api`` package to the two most popular logging frameworks, Log4j2 and
+Logback.
+
+Example
+~~~~~~~
+
+This example shows how to set up your logger using Maven. Click the
+tab corresponding to the logging framework you would like to use in your project.
+
+.. tabs::
+
+   .. tab:: Logback
+      :tabid: Logback-binding
+
+      Add the following dependencies to your ``pom.xml`` file.
+
+      .. code-block:: xml
```

Comment:
Are there plans to add gradle style examples  - as per the quickstart?

eg:

```
dependencies {
   implementation 'org.slf4j:slf4j-api:1.7.30'
   implementation 'ch.qos.logback:logback-classic:1.2.3'
}
```

Reply:
I did not plan to add them, but I can add them now. I was hesitant to add them as I didn't want two layers of tabs, but gradle users will probably appreciate the information. Addition: